src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-courier-pop-up/ubs-admin-tariffs-courier-pop-up.component.ts

```diff
@@ -81,9 +82,8 @@ export class UbsAdminTariffsCourierPopUpComponent implements OnInit, OnDestroy {
       .pipe(takeUntil(this.destroy))
       .subscribe((res) => {
         this.couriers = res;
-        this.couriersName = res
-          .map((it) => it.courierTranslationDtos.filter((ob) => ob.languageCode === 'ua').map((item) => item.name))
-          .reduce((acc, val) => acc.concat(val), []);
+        this.couriersName = res.map((it) => it.courierTranslationDtos.map((el) => el.name)).flat(2);
```

Comment:
I feel like there should not be multiple `courierTranslationDtos` for a single courier, maybe you can get it here with array destructuring or courierTranslationDtos[0], it would be great to avoid nested maps since it is hard to read.